drivers/note/note_driver.c

```diff
@@ -469,190 +553,290 @@ void sched_note_start(FAR struct tcb_s *tcb)
   length = SIZEOF_NOTE_START(0);
 #endif
 
-  /* Finish formatting the note */
-
-  note_common(tcb, &note.nsa_cmn, length, NOTE_START);
+  for (; *driver; driver++)
```

Comment:
```suggestion
  for (driver = g_note_drivers; *driver; driver++)
```
and remove assignment at line 531

Reply:
done